pkg/authorization/cache/policy_cache_test.go

```diff
@@ -0,0 +1,96 @@
+package cache
+
+import (
+	"testing"
+	"time"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	testregistry "github.com/openshift/origin/pkg/authorization/registry/test"
+)
+
+func TestPolicyGet(t *testing.T) {
+	policyRegistry := testregistry.NewPolicyRegistry(testPolicies(), nil)
+	bindingRegistry := testregistry.NewPolicyBindingRegistry(testBindings(), nil)
+
+	policyCache := NewPolicyCache(bindingRegistry, policyRegistry)
+	policyCache.Run(1 * time.Second)
+
+	time.Sleep(50 * time.Millisecond)
```

Comment:
Generally you would instead call "RunOnce" or something to drive one cycle (or drive the individual components).  You need to be running and ensuring your things shut down, so the structure of your cache should support running and stopping.